src/platform/unix/ushellcontextmenu.pas

```diff
@@ -642,29 +642,33 @@ procedure TShellContextMenu.CreateActionSubMenu(MenuWhereToAdd:TComponent; aFile
   if ContextMenuActionList.Count>0 then
     LocalInsertMenuSeparator;
 
-  // If the external generic viewer is configured, offer it.
-  if gExternalTools[etViewer].Enabled then
+  // If the default context actions not hidden
+  if DefaultContextActions then
```

Comment:
It should be `gDefaultContextActions` here and below.

Reply:
Fix it! 👍